libs/cart/src/selectors/cart-order/cart-order.selector.ts

```diff
@@ -22,13 +23,14 @@ export interface DaffCartOrderMemoizedSelectors<
 
 const createCartOrderSelectors = <
   T extends DaffCart = DaffCart,
-  V extends DaffCartOrderResult = DaffCartOrderResult
+	V extends DaffCartOrderResult = DaffCartOrderResult,
+	U extends DaffCartItem = DaffCartItem
```

Comment:
You need to add this generic to `getCartOrderSelectors` as well.

Reply:
done